src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -302,6 +309,133 @@ static StaticInitializers s_static_initializers(s_output_dir);
 
 #define TI(x) type_index(typeid(x))
 
+#ifdef _MSC_VER
+// MSVC (and clang on Windows via clang-cl) instantiates a template
+// even when we know that it will be instantiated in cpu_emitter.cpp.
+// It leads to linker error: 'error: duplicate symbol'.
+extern template void runtime::cpu::CPU_Emitter::EMITTER_DECL(op::Add);
```

Comment:
Can we alphabetize this list?

Reply:
And move to a header file. Too much clutter here.

Reply:
@rkimballn1 @nmostafa Well, does `cpu_emitter_templates.hpp` sound good?